sharktuner/sharktuner/common.py

```diff
@@ -202,6 +212,38 @@ class AttentionOpInfo:
     k2_dims: list[int]
 
 
+@dataclass
+class LLVMGPUVectorDistributeContractionKnobs(KnobAssignment):
+    # Z3 numeric selections
+    m: int
+    n: int
+    k: int
+    wg_x: int
+    wg_y: int
+    wg_z: int
+    sg_m_cnt: int
+    sg_n_cnt: int
+    intrinsic_mn: int
+    intrinsic_k: int
+    subgroup_m: int
+    subgroup_n: int
+    subgroup_k: int
+    num_subgroups: int
```

Comment:
Do we need this? I think this can be a method that multiplies `sg_m_cnt` and `sg_n_cnt`